src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -92,6 +92,13 @@ public function load(array $configs, ContainerBuilder $container)
             $dir = dirname($reflection->getFileName()).'/Resources/views';
             if (is_dir($dir)) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $bundle);
+
+                $classMock = new $class();
```

Comment:
The issue is that you still would not have access to actual instances of the bundles (so this would not solve https://github.com/symfony/symfony/pull/19586#r94117685).